blog/_posts/2017-05-23-whats-new-scala-native.md

```diff
@@ -0,0 +1,260 @@
+  ```
+* Scala Native can now read environment variables using `System.getEnv`,
+  thanks to [@jonas][@jonas]' efforts in [#606][#606].
+* `stdin` and `stdout` can reliably be read from and written to, thanks to
+  [@fduraffourg][@fduraffourg] and [@densh][@densh] in [#622][#622] and
+  [#659][#659].
+
+### Bugfixes
+
+* A bugfix has been contributed by [@jonas][@jonas] to `String.replace`,
+  fixing one broken benchmark at the same time. His work can be found in
+  [#616][#616].
+* `System.nanoTime` was fixed by [@brad-rathke][@brad-rathke] in [#549][#549].
+* [@xuwei-k][@xuwei-k] fixed `nativeAvailableDependencies` which didn't work
+   in the `test` configuration in [#565][#565].
+
+### Improvements to the tooling and integration
+
+* [@MasseGuillaume][@MasseGuillaume] and [@densh][@densh]
+  worked on refactoring Scala Native's sbt plugin to make it more idiomatic in
+  [#568][#568] and [#630][#630], which lead to fixing [#562][#562].
+* Follow up fixes were contributed by [@jonas][@jonas], and include [#639][#639]
+  and [#653][#653].
+
+### Preparing for a better garbage collector
+
+In this release, [@LukasKellenberger][@LukasKellenberger] introduced in
+[#539][#539] a new setting in the sbt plugin that lets users select what
+implementation of the garbage collector should be used. Currently, BoehGC
+and no GC are supported.
+
+This work was done in preparation for [the improved GC that will be shipped
+with Scala Native 0.3][#726]!
+
+
+## A community effort
+
+As shown, many of the improvements that were brought by Scala Native 0.2 have
+been contributed by members of the vibrant community that is developing itself
+around Scala Native. Thank you!
+
+The combination of these improvements was enough to get a prototype of
+`scalafmt` running on Scala Native by [@olafurpg][@olafurpg], showing a blazing
+fast startup time!
+
+<blockquote class="twitter-tweet" data-conversation="none" data-lang="en">
+  <p lang="en" dir="ltr">
+    <a href="https://twitter.com/adriaanm">@adriaanm</a>
+    <a href="https://twitter.com/den_sh">@den_sh</a>
+    <a href="https://twitter.com/scala_native">@scala_native</a> OMG IT WORKS :D 30x faster!
+    <a href="https://t.co/M7V9udU5bT">pic.twitter.com/M7V9udU5bT</a>
+  </p>
+  &mdash; Ólafur Páll Geirsson (@olafurpg)
+  <a href="https://twitter.com/olafurpg/status/857559907876433920">April 27, 2017</a>
+</blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+## What to expect for Scala Native 0.3?
+
+The plans for the next release of Scala Native include a new garbage collector,
+a better integration with sbt and more addition to the standard library.
+
+### Improved garbage collector
+
+The first releases of Scala Native use Boehm GC. A new and improved garbage
+collector has been under development by [@LukasKellenberger][@LukasKellenberger]
+and will be presented at Scala Days during Denys' talk. Stay tuned!
```

Comment:
I'd also link to Denys's talk and post the date of Scala Days, so that people who aren't closely following everything in the Scala universe know how soon they'll hear more about the new GC.